File: Lecture6/flask_api/restaurant.py
from pynamodb.models import Model
from pynamodb.attributes import UnicodeAttribute, NumberAttribute

class Restaurant(Model): # inheritance
    class Meta:
        table_name = "Restaurant"
        region = 'us-east-1'
    resName = UnicodeAttribute(hash_key=True)
    city = UnicodeAttribute()
    category1 = UnicodeAttribute()
    category2 = UnicodeAttribute()
    rating = NumberAttribute()
    image_url = UnicodeAttribute()
    priceRange = UnicodeAttribute()

File: Lecture3/lecture_3/lib/core/model/restaurant.dart
class Restaurant {
  String resName;
  String priceRange;
  String cuisineType1;
  String cuisineType2;
  double rating;
  String resImage;
  String address;

  Restaurant(String resName, String priceRange, String cuisineType1,
      String cuisineType2, double rating, String resImage, String address) {
    this.resName = resName;
    this.priceRange = priceRange;
    this.cuisineType1 = cuisineType1;
    this.cuisineType2 = cuisineType2;
    this.rating = rating;
    this.resImage = resImage;
    this.address = address;
  }
}
